### PR TITLE
feat(insights): make the page actionable — verdict banner, context, run stats

### PR DESCRIPTION
**Pull request by mthines**
## Why

`/insights` (still flagged off) surfaced raw diagnostics with no "so what" — a failure count with nowhere to click, an "unattributed" client with no explanation, a truncated scope list with no way to see the rest, and a Runs ledger nobody could interpret. Separately, the page's health figures mixed in the dashboard's own page-loads as if they were agent traffic, and there was no way to see whether an agent's usage was trending better or worse over time.

A follow-up review of the assembled page found the deeper problem: it still read as instrumentation rather than as something that changes what a reader does next. Its headline figure could not move, its two leaderboards were dead ends, and turning the flag on stranded onboarding on a page nothing links to.

## What changed

**Making the page legible (first pass)**

- Add an at-a-glance `HealthSummary` verdict banner above the diagnostic panels, reusing the already-fetched usage summary
- Make Friction/Who's-reading/Scope-consumption actionable: each failure names its dominant client+scope context, "unattributed" gets an explanatory tooltip, and the scope list expands instead of truncating
- Surface each run's read/write/scope counts inline in `RunsList` instead of only after expanding, and reframe the section as an audit trail rather than a health signal
- Exclude dashboard-originated calls from `HealthSummary`/`UsageHealth` (`excludeDashboardReads`) — browsing the Explorer yourself isn't "how your agents are actually using them"; `AgentBreakdown`'s "who is reading" keeps the full set since showing that split is its job
- Add a shared "Agent activity" range picker (`24h`/`7d`/`30d`/`90d`) driving `HealthSummary`/`UsageHealth`/`AgentBreakdown` from one fetch, plus week-over-week trend chips (`healthTrend`)

**Making it actionable (review pass)**

- **The verdict is two-dimensional now.** `healthVerdict` grades reliability *and* `readCoverage`'s records-per-read and reports the worse of the two, with the banner naming which drove it. Success rate alone never moved: `usage_events.outcome` is `ok | cap_exceeded | rate_limited | permission_denied | error`, none of which means "the agent asked for lore and got nothing" — an empty-scope read is `ok`. So a healthy account read "100% of calls succeeded" every day in the page's largest, first slot, while the signal a reader came for sat three columns into the section below. Coverage counts record-bearing tools only (`memory.list`/`read`/`search`), since `memory.write`/`org.*` are structurally `record_count: 0`; a window with no reads reports an absence ("no reads"), never a `0.0` that would read as failure.
- **`healthTrend` needs a real baseline** — 20 calls in the previous window, not merely non-zero. One prior call against a thousand rendered "+99,900%".
- **Both leaderboards click through into the Explorer.** A `ScopeConsumption` row links to `/lore?scope=SCOPE`, a `HotColdLore` row to `/lore?scope=SCOPE&q=KEY`. The `unattributed` row stays inert — it has no scope to narrow to, and linking it would show a different set of lore than the bar measures. Two encoding constraints: `scope` is deliberately not a `?filters=` dimension, and the Explorer has no `?lesson=` param (its detail sheet is local state), so scope + search is the closest honest target.
- **Insights inherits Overview's onboarding duties along with its home slot.** The flag removes `/overview` from nav, and `/overview` was the only host of `PendingInvitesBanner`/`OnboardingChecklist`/`GithubAppTeaser` — and the only caller of `buildOnboardingSteps({ autoGenerateToken: true })`, which mints a brand-new user's first API token. A flag-on signup landed on an analytics page with no token, no setup instructions, and no way to find either. All four move onto `insights/page.tsx`, the root redirect points at `/insights`, and Insights takes the first nav slot so the rail matches the claim. All three surfaces self-hide once irrelevant, so an established account sees the page unchanged.
- **`RangePicker` takes a `label`**, so the page's two independent radiogroups stop sharing one accessible name.
- The banner and the Agent activity caption state that dashboard browsing is excluded rather than leaving it implicit.

## How to verify

- `pnpm nx typecheck web && pnpm nx lint web && pnpm nx test web`
- `cd packages/web && npx vitest run --config vitest.storybook.config.ts --changed=main`

Visual baselines for `HealthSummary` and `InsightsPage` were regenerated (`-u`) for the banner's new two-figure layout; two new `HealthSummary` stories cover the coverage-driven verdict and the write-only window.

---
_Generated by [Claude Code](https://claude.ai/code/session_01VxLF67tz1z63us1ZuG4zfN)_

### CODE REVIEW

File: packages/web/src/components/insights/InsightsPage.tsx [hunk 1]
```diff
@@ -108,6 +115,10 @@ export function InsightsPage() {
         </p>
       </div>
 
+      {!isLoading && !isError && data && usageByTool.length > 0 && (
```
[the remark this answers is not in the document]
Comment (mthines) [reply packages/web/src/components/insights/InsightsPage.tsx]: Addressed in 5e6d5a7 — dropped the `usageByTool.length > 0` guard. Also fixed the `Empty` story's `/usage` mock (it had a nonzero `summary.total_events` despite empty `by_tool`, which would have made this render a fabricated "healthy" banner instead of the empty state) and regenerated the screenshot baseline.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: packages/web/src/components/settings/RunsList.tsx [hunk 2]
```diff
@@ -162,20 +162,31 @@ function RunRow({
...
+            {run.correlation_id}
+          </code>
+          <span className="shrink-0 text-xs text-[var(--color-content-tertiary)]">
+            {new Date(run.last_seen).toLocaleString()}
+          </span>
+        </div>
+        {/* The stats that used to be visible only after expanding — enough to
+            tell an interesting run (lots of writes, many scopes) from a boring
+            one (a handful of reads) without opening it. */}
+        <div className="ml-7 flex flex-wrap items-center gap-x-3 gap-y-0.5 font-mono text-[10px] text-[var(--color-content-tertiary)]">
```
[the remark this answers is not in the document]
Comment (mthines) [reply packages/web/src/components/settings/RunsList.tsx]: Addressed in a57da0e — added `aria-hidden` to the stats row.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by mthines** [comment]
The `Web — Vercel preview (changed) / Deploy Vercel preview` check failed on `4cd98c0` with:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/runner/work/lorekit/lorekit/scripts/ci/web-preview-comment.mjs'
```

Not this PR's failure — the actual Vercel deploy/alias succeeded; only the follow-up "post preview URL" step broke. This PR's base predated [PR #636](https://github.com/mthines/lorekit/pull/636) ("Extract web-preview comment renderer to a testable module"), which added `scripts/ci/web-preview-comment.mjs` on `main`. Since `pull_request`-triggered workflows read the workflow YAML from `main` but this job checks out the PR head commit, the newer workflow definition referenced a script this branch didn't have yet.

Fix: merged `origin/main` into this branch (`7ceea5c`) to pick up the script. Clean merge, no conflicts. Re-validated `typecheck`/`lint` on the affected projects post-merge (0 errors).

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by mthines** [comment]
## `Web — Vercel preview (changed)` is red on `d65facf` — not this PR's failure

Standing down on this check with no code change. What is failing, why it isn't this branch's, and the one patch I'd propose for the diagnostic gap it exposed.

### What is failing

`Web — Vercel preview (changed) / Deploy Vercel preview` — specifically the `Deploy Vercel preview` step, which exits **124** after exactly `900032 ms`: the composite action's `DEPLOY_TIMEOUT: 15m` on `timeout … vercel deploy --prebuilt`.

It is **not part of the CI gate**. `CI Summary` — the single required check — lists `needs: [changes, check, plugin, integration, web-test, migration-order, deploy-scope, preview-filter, bench-tests, edge-typecheck]`, and `web-preview` is deliberately absent. `CI Summary` is green on this head, as are `Detect changes`, `Typecheck, Test & Lint (affected)` and `Web — Storybook interaction + visual tests (changed)`.

### Why it isn't this PR's

1. **The build succeeds.** The runner-side `vercel build` completes — `✓ Generating static pages (51/51)`, `Build Completed in .vercel/output`, `/insights` at 16.2 kB / 339 kB First Load JS. Only the *prebuilt upload's* polling never reaches Ready. Once `.vercel/output` exists on disk, application code has no remaining influence on whether Vercel marks the deployment Ready.
2. **The same author deployed cleanly 5× on this branch**, most recently `c1bfff5` at 15:04 UTC with a 2-minute deploy step. Every commit here is authored `Claude <noreply@anthropic.com>`, so the workflow's own hint — "the commit author's email can't be matched to a GitHub account with access to the project" — is refuted by this branch's own history.
3. **`4cd98c0`'s earlier red was a different step** (`Comment preview URL on the PR`), with the deploy itself green in 2m21s. Different failure, not a recurrence.
4. **The same signature exists on an unrelated PR.** [`web-preview.yml` run 479](https://github.com/mthines/lorekit/actions/runs/33538001903) — 2026-09-01, PR #629 — failed identically at ~17 minutes.

### The re-run was spent, and it failed the same way

[Run 33676149673](https://github.com/mthines/lorekit/actions/runs/33676149673) (`workflow_dispatch` of `web-preview.yml`, `force: true`, redeploying the unchanged head): `Deploy Vercel preview` ran 19:55:14 → 20:12:08 and exited 124 again. Two consecutive attempts, ~4.5 hours apart, both with a successful build.

The CLI emitted **no progress output at all** across those 15 minutes — no `Inspect:` line, no `Queued`, no `Building`. Consistent with a deployment that never leaves Vercel's queue (an org concurrency or deployment-rate ceiling would present exactly this way), which is an account/project-side matter rather than something reachable from this diff.

### The diagnostic gap this exposed — proposed patch, not applied here

`.github/actions/vercel-preview-deploy/action.yml` intends to capture the CLI's exit code and report the reason itself:

```bash
# NOT `set -e`: we capture `vercel`'s exit code and report the reason
# ourselves rather than aborting on it.
set -uo pipefail
out="$(timeout --kill-after=30s "$DEPLOY_TIMEOUT" vercel deploy --prebuilt --token="$VERCEL_TOKEN" 2>"$err_file")"
code=$?
```

That path is dead. GitHub runs a composite `run:` step under `bash --noprofile --norc -e -o pipefail`, and `set -uo pipefail` does not clear the inherited `-e` — so the failing assignment aborts the step before `code=$?`. Nothing is echoed, and `vercel-deploy-error.txt` is never written, which is why the sticky comment fell back to the generic "The Vercel deployment did not complete. See the workflow logs for the CLI output" instead of the specific 15m-timeout message the action composes. The stderr the reporting path exists to quote is also lost with the temp file.

The minimal fix is to make the step's own intent explicit rather than relying on the inherited shell flags:

```yaml
      shell: bash --noprofile --norc {0}
```

or, keeping the default shell, `set +e` alongside the existing `set -uo pipefail`.

I have deliberately **not** applied that here — it is CI-infrastructure work unrelated to this PR's subject (the Insights page), and widening the diff to carry it would make this branch harder to review, not easier. It's a one-line change whenever someone wants it, and it would have turned this investigation into a single log line.

### What I did not do

Raising `DEPLOY_TIMEOUT` would hide a deployment that genuinely never completes, and the guard is there precisely to stop a blocked deploy from running to the 6-hour job default. There is no fix inside this PR's scope, and no fix elsewhere in the repo to port into it.

---
_Generated by [Claude Code](https://claude.ai/code)_
